### ISSUE TITLE
HA core-2021.9.x icon broken integration page

### ISSUE THREAD

**Issue by CaptainKarma** [Danielhiversen/home_assistant_adax#42]
Hi

The integration seems to have broken in .9.  The Adax icon is not displaying properly on the integration page, was working fine in the .8.

I can still get stats from my heaters, so I think its just a cosmetic thing.

Best Regards

**Comment by thefunkygibbon** [comment]
yeah, would be nice if this was updated to fix this from an aesthetic perspective.   also slightly concerning if an update to HA breaks this integration if its not been updated in a while.

**Comment by stgrdk** [comment]
I noticed this to, and also the name of the integration was my account id, so i renamed this to Adax, and after that the icon showwd up again.

**Comment by nwgat** [comment]
> I noticed this to, and also the name of the integration was my account id, so i renamed this to Adax, and after that the icon showwd up again.

rename fixes it :)
